src/lib/history.ts

```diff
@@ -0,0 +1,29 @@
+import { WeeklyArticleLog, WeeklyArticleHistory } from '../components/ContributionsEpicTypes';
+
+export const getMondayFromDate = (date: Date): number => {
```

Comment:
I wonder if there's a name here we could use to better convey that we're returning a timestamp? (and not a `Date`)?

Reply:
I think the return type is clear we're getting a number not a date, a number in this context can only indicate it's a timestamp. 
I'd be much more keen on renaming the function itself to be more explicit but I can't think of a decent name for it so will leave as is until such name comes up. To really be descriptive, it would be called `getDaysSinceEpochRoundedToThisWeeksMonday` but OMG we can't do that

Reply:
We could break this function in 2, to be able to name them better, but each individually would have little to no value and I'm against doing that just for naming  reasons.

Reply:
Finally - given the difficulty naming this function, at least we keep the same named used in Frontend.